tests/tiup-cluster/topo/full_scale_in_prometheus.yaml

```diff
@@ -0,0 +1,3 @@
+monitoring_servers:
+  - host: n2
+    rule_dir: /tmp/local/prometheus
```

Comment:
I think this is the root cause that the test failed: you don't have this path on n2. So just delete this line, and the default rules will be used.

Reply:
And BTW, why we should scale-out a prometheus and scale-in? there is already a prometheus in the cluster. 

Maybe we can just find the existed (not in delete set) prometheus and reload them only.